tools/kontaktdaten.py

```diff
@@ -3,24 +3,27 @@
 import re
 
 from email.utils import parseaddr
+from tools.exceptions import ValidationError, MissingValuesError
+from tools import Modus
 
 
-class ValidationError(Exception):
-    pass
-
-
-def get_kontaktdaten(filepath):
+def get_kontaktdaten(filepath: str, mode: Modus):
     """
     Lade Kontaktdaten aus Datei.
 
     :param filepath: Pfad zur JSON-Datei mit Kontaktdaten.
+    :param modus: Unterschiedliche Kriterien ob key / values vorhanden sein müssen
     :return: Dictionary mit Kontaktdaten
+
+    :raise: ValidationError: Wird geworfen wenn eine Datei ungültige Values besitzt
+    :raise: MissingValuesError: Wird geworfen, wenn benötigte Key / Values nicht da sind
     """
 
     try:
         with open(filepath, encoding='utf-8') as f:
             try:
                 kontaktdaten = json.load(f)
+                check_kontaktdaten(kontaktdaten, mode)
```

Comment:
Das hier auszuführen ist problematisch, bzw. es macht die CLI kaputt. Bisher dürfen in `kontaktdaten.json` Sachen fehlen; die CLI fragt dann nach den fehlenden Sachen. Dass da Sachen fehlen ist auch durchaus üblich, wenn man erst einmal einen Code anfragt, da beim Code-anfragen nur bestimmte Felder geschrieben werden. Die restlichen fehlen erstmal und werden dann angefragt, wenn man eine Suche startet. Mit deiner Änderung würde aber bereits das Laden der `kontaktdaten.json` fehlschlagen wegen fehlenden Feldern.

Reply:
Die richtige Stelle das auszuführen wäre am Anfang von `run_search` und `gen_code`.